components/video/VideoPlayerView.bs

```diff
@@ -1356,6 +1357,9 @@ function onKeyEvent(key as string, press as boolean) as boolean
         m.PreloadTrickplayImagesTask.method = "REMOVE"
         m.PreloadTrickplayImagesTask.control = TaskControl.RUN
 
+        currentTimeTimer = m.clock.findNode("currentTimeTimer")
+        if isValid(currentTimeTimer) then currentTimeTimer.control = TimerControl.STOP
```

Comment:
Clock.bs::25

TimerControl.START

😄

Reply:
Sorry... I was probably too cryptic 😆 

File: Clock.bs
Line: 25
Not using the new Enum 😄